apis/cluster/v1/membercluster_types.go

```diff
@@ -22,6 +22,7 @@ import (
 // +kubebuilder:printcolumn:JSONPath=`.status.resourceUsage.available.memory`,name="Available-Memory",type=string
 // +kubebuilder:printcolumn:JSONPath=`.status.resourceUsage.allocatable.cpu`,name="Allocatable-CPU", priority=1, type=string
 // +kubebuilder:printcolumn:JSONPath=`.status.resourceUsage.allocatable.memory`,name="Allocatable-Memory", priority=1, type=string
+// +kubebuilder:printcolumn:JSONPath=`.status.agentStatus[?(@.type=="MemberAgent")].lastReceivedHeartbeat`,name="Last-Heartbeat-Member-Agent",type=string
```

Comment:
Should this be the last one?

Reply:
$k get ns -o wide
NAME              STATUS   AGE
calico-system     Active   235d
default           Active   235d
kube-node-lease   Active   235d

Reply:
which is better? AGE or the plain time?